lxd/sys/apparmor.go

```diff
@@ -71,9 +71,9 @@ func (s *OS) initAppArmor() []cluster.Warning {
 
 	/* Detect AppArmor confinment */
 	profile := util.AppArmorProfile()
-	if profile != "unconfined" && profile != "" {
+	if profile != "unconfined" && profile != "" && !strings.HasSuffix(profile, "(unconfined)") {
```

Comment:
Please could you add a comment here explaining why it is checking for '(unconfined)' in the profile name and where that comes from for future readers. Thanks

Reply:
Done in https://github.com/canonical/lxd/pull/12713/commits/04a445bb360b728ea6876ba086e558ec1d0de832

Reply:
Thanks! this is great.